features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/services/login/token_callback.jag

```diff
@@ -1,64 +1,119 @@
 <%
     var log = new Log("Token Request Function");
+    var state = request.getParameter("state","UTF-8");
+    log.debug("state: " + state);
+    if ( state !== null ) {
```

Comment:
@ruwiniwj Can you please pull these improvements into your PR https://github.com/wso2/carbon-apimgt/pull/6466, Basically we would need to refactor the token_callback.jag file and split the login callback handler and logout callback handler into two separate files.
* Would need to add new routing in jaggery conf
* Would need to change the callback_url parameter in DCR call to accept two callback endpoints in login and logout requests